src/blueapi/core/bluesky_types.py

```diff
@@ -24,7 +24,7 @@
     Triggerable,
     WritesExternalAssets,
 )
-from bluesky.utils import MsgGenerator
+from dls_bluesky_core.core import MsgGenerator
```

Comment:
```suggestion
from bluesky.utils import MsgGenerator
```

Reply:
revert

Reply:
I specifically added this yesterday, adjusting imports and versions 😠

Reply:
`dls-bluesky-core` is deprecated though